dist/idf-mobilite-card-editor.js

```diff
@@ -164,6 +172,12 @@ class IDFMobiliteCardEditor extends LitElement {
                       .configValue=${"exclude_lines_ref"}
                       @input="${this._valueChanged}"
                     ></ha-textfield>
+                    <ha-textfield
```

Comment:
Can this `exclude mission` text field be shown in the card editor only if the user chooses RER type (if it's doable)?